### DISCUSSION TITLE
What is that 1-6 at the bottom of the screen?

### DISCUSSION THREAD

**Discussion by HippoDan** [EdgeTX/edgetx#5050] [Q&A]
There is a 1-6 at the bottom of the main screen.  Mine always shows 6.  No settings I change seem to affect it.  What is it for?

**Comment by mhk1058** [comment]
That's showing you which position your 6 position switch is in. Not sure why yours is always at 6, mine defaults back to 1 on start up.. but maybe it's dependant on how you set your model up, or maybe you have a flight mode set up.

**Reply by HippoDan** [reply]
Weird.  6Pos switch changes nothing, neither does flight mode.  V2.10 TX16s Mk2.  I'll play around with it some more.

**Comment by HippoDan** [comment]
Okay, I went back to the settings and tried everything I could think of.  Turning off the checkbox next to the 6POS setting under pots fixed it.  So, my question now is, what are those checkboxes supposed to mean?

Thanks.

**Reply by mhk1058** [reply]
I think it inverts the direction... which would explain why yours started at position 6, rather than 1.

**Comment by pfeerick** [comment]
End checkbox in hardware pots section inverts direction.

The 1-6 boxes represent the position of the 6pos switch, just like how the
slider bars represent position of those controls. And just like with those
controls, won't *do* anything if not assigned to a input/mix/special
function , etc. if you press the buttons, and it doesn't change which
number the box is on, you need to do calibration again (on hardware page),
as 6pos is uncalibrated. I.e. when prompted to centre sticks, also centre
your sliders and pots. And when prompted to move the sticks, also move the
pots and sliders full range, and press each button of the 6pos, with a
pause in between each change. I.e. I do 23456.54321 with roughly half
second pause between presses.

**Reply by HippoDan** [reply]
Thanks for the clarification.   It's calibrated properly,  just pegs the switch at 6 when reversed.  No problem now that I know what it's for.

**Comment by philmoz** [comment]
I don't think inversion makes sense for the 6POS switches - probably the invert option should be hidden in the UI in this case.

**Reply by pfeerick** [reply]
For the physical six button no, but it may be relevant for rotary 6POS knobs.
